File: problemset6/sentimental-cash/cash.py
while True:
    try:
       change = float(input("Change owed: "))
       if(change > 0):
          break
    except:
      print("input a integer")

coins = round(change * 100)

count = 0
while coins > 0:
    if coins >= 25:
        coins -= 25
        count += 1
    elif coins >= 10:
        coins -= 10
        count += 1
    elif coins >= 5:
        coins -= 5
        count += 1
    else:
        coins -= 1
        count += 1

print(count)

File: problemset1/cash/cash.c
#include<stdio.h>
#include<cs50.h>

int add(int a, int b);
int main(void)
{
  int A;
  do
  {
    A = get_int("Change owed: ");
  }
  while(A < 0);

    int quarters = 0;
    while(A >= 25)
    {
      quarters++;
      A = A -25;
    }

    int tencents = 0;
    while(A >= 10)
    {
      tencents ++;
      A = A - 10;
    }

    int fivecents = 0;
    while(A >= 5)
    {
      fivecents ++;
      A = A - 5;
    }

    int pennies = 0;
    while(A >= 1)
    {
      pennies++;
      A = A -1;
    }

    int coins = (quarters + tencents + fivecents + pennies);
    printf("%i\n", coins);
}
